test/integration/consul-container/libs/assert/service.go

```diff
@@ -100,20 +100,36 @@ func ServiceLogContains(t *testing.T, service libservice.Service, target string)
 func AssertFortioName(t *testing.T, urlbase string, name string) {
 	t.Helper()
 	var fortioNameRE = regexp.MustCompile(("\nFORTIO_NAME=(.+)\n"))
-	var body []byte
+	client := &http.Client{
+		Transport: &http.Transport{
+			DisableKeepAlives: true,
```

Comment:
To restart the connection after route is changed, otherwise the connection is till to the previous instance